electroncash/blockchain.py

```diff
@@ -85,27 +85,27 @@ def target_to_bits(target):
 NULL_HASH_HEX = NULL_HASH_BYTES.hex()
 
 def serialize_header(res):
-    s = int_to_hex(res.get('version'), 4) \
-        + rev_hex(res.get('prev_block_hash')) \
-        + rev_hex(res.get('merkle_root')) \
-        + int_to_hex(int(res.get('timestamp')), 4) \
-        + int_to_hex(int(res.get('bits')), 4) \
-        + int_to_hex(int(res.get('nonce')), 4)
+    s = bitcoin.int_to_hex(res.get('version'), 4) \
+        + bitcoin.rev_hex(res.get('prev_block_hash')) \
+        + bitcoin.rev_hex(res.get('merkle_root')) \
+        + bitcoin.int_to_hex(int(res.get('timestamp')), 4) \
+        + bitcoin.int_to_hex(int(res.get('bits')), 4) \
+        + bitcoin.int_to_hex(int(res.get('nonce')), 4)
     return s
 
 def deserialize_header(s, height):
     h = {}
     h['version'] = int.from_bytes(s[0:4], 'little')
-    h['prev_block_hash'] = hash_encode(s[4:36])
-    h['merkle_root'] = hash_encode(s[36:68])
+    h['prev_block_hash'] = bitcoin.hash_encode(s[4:36])
+    h['merkle_root'] = bitcoin.hash_encode(s[36:68])
     h['timestamp'] = int.from_bytes(s[68:72], 'little')
     h['bits'] = int.from_bytes(s[72:76], 'little')
     h['nonce'] = int.from_bytes(s[76:80], 'little')
     h['block_height'] = height
     return h
 
 def hash_header_hex(header_hex):
-    return hash_encode(Hash(bfh(header_hex)))
+    return bitcoin.hash_encode(bitcoin.Hash(bytes.fromhex(header_hex)))
```

Comment:
This was changed from `bfh()` to `bytes.fromhex()`. Please check that was intended and see a related comment below.

Reply:
This is to avoid using an alias that was previously indirectly import via `from .bitcoin import *`. 
See https://github.com/Bitcoin-ABC/ElectrumABC/blob/master/electroncash/util.py#L453
and https://github.com/Bitcoin-ABC/ElectrumABC/blob/master/electroncash/bitcoin.py#L40